scrapers/Pornditos.yml

```diff
@@ -19,13 +24,44 @@ xPathScrapers:
         selector: //div[@id="js-small-thumb"]/img/@src
         postProcess:
           - replace:
-              - regex: "http://sm.members.khcdn.com/shoots/([^/]+)/([^/]+)/rollover.*"
-                with: "http://x-images5.bangbros.com/$1/shoots/$2/members/450x340.jpg"
+              - regex: 'http://sm\.members\.khcdn\.com/shoots/([^/]+)/([^/]+)/rollover.*'
+                with: "http://x-images5.bangbros.com/$1/shoots/$2/big_trailer.jpg"
+              - regex: '5\.bangbros\.com/cuchimami'
+                with: "4.bangbros.com/cuchimami"
       Date:
         selector: $info/p[starts-with(text(),"Added to")]/text()[last()]
         postProcess:
           - replace:
               - regex: on\s
                 with:
           - parseDate: Jan 2, 2006
-# Last Updated August 08, 2021
+
+  culionerosScraper:
+    common:
+      $info: //h2[@class="h2-trailer-block"]
+    scene:
+      Title: $info//span[@class="h2-trailer-style"]/text()
+      Details: 
+        selector: //p[@class="para-class t-para"]/text() 
+        postProcess:
+          - replace:
+              - regex: \n
+                with: " "
+      Performers:
+        Name: $info//a[contains(@href, "/models/")]/text()
+      Studio:
+        Name: $info//a[contains(@href, "/latest-videos/")]/text()
+      Image:
+        selector: //img[contains(@src, "/shoots/")][1]/@src # unsure why index 0 is not working
+        postProcess:
+          - replace:
+              - regex: '(http://x-images\d\.bangbros\.com/[^/]+/shoots/[^/]+).*'
+                with: "$1/big_trailer.jpg"
+      Date:
+        selector: $info//b[@class="t-font"]/text()
```

Comment:
The selector matches an extra text (the comma that separates the performers)
A more explicit selector should work fine
```yaml
        selector: $info//b[@class="t-font"][not(a)]/text()
```